index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="ja">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>課題②-1</title>
+    <style>
+      label {
+        display: block;
+        float: left;
+        width: 100px;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>FizzBuzz問題</h1>
+    <label for="fizz">FizzNum：</label>
+    <input type="text" name="fizz" id="fizz" placeholder="整数値を入力してください" /><br />
+    <label for="buzz">BuzzNum：</label>
+    <input type="text" name="buzz" id="buzz" placeholder="整数値を入力してください" /><br />
+    <input type="button" name="button" id="button" value="実行" />
+    <h3>【出力】</h3>
+    <div id="out"></div>
+    <script>
+      'use strict';
+
+      // 入力値のチェック関数
+      function chkNum(num) {
```

Comment:
関数の宣言をクリックイベント内で宣言していたので、外に移動しました。

Reply:
この関数の名前について。
気持ちはとてもわかるのですが、あえて厳しいことを言うと…
「入力値のどう言った点をチェックするのか」
「チェックした結果、何が戻り値として戻されるのか」
と言うことがいまいちわからない名前になってしまっています。
この関数は入力値が入力されており、かつ整数であることを boolean で返すのですから以下のような名前はいかがでしょうか？

```suggestion
      function isNotNullAndInteger(num) {
```